source/databases-collections.txt

```diff
@@ -58,7 +58,9 @@ Alternatively, right-click a collection and click
 in an array.
 
 Opening collection documents provides a **read-only** view of your data.
-To modify your data using |vsce|, use a JavaScript Playground.
+To modify your data using |vsce|, use a :ref:`JavaScript Playground
+<vsce-crud>` or launch a shell by right-clicking your active deployment
+in the left-side navigation.
```

Comment:
Off-ticket improvement here.

Reply:
nit: the navigation might not be on the left side -- I moved it to the right side. Consider: ... in the MongoDB view in the Activity Bar. Looks like this is what we use [elsewhere](https://docs.mongodb.com/mongodb-vscode/connect#disconnect-from-a-deployment).

Reply:
Done.